packages/apps/manufacturing/src/screens/operationOrder/OperationOrderListScreen.js

```diff
@@ -51,9 +51,8 @@ function OperationOrderListScreen({navigation}) {
     state => state.operationOrder,
   );
   const {workCenterList} = useSelector(state => state.workCenters);
-  const {machineList} = useSelector(state => state.machines);
   const [workCenter, setWorkCenter] = useState(null);
-  const [machine, setMachine] = useState(null);
+  const [machine] = useState(null);
```

Comment:
Do not leave useState without setter, this is useless as value cannot be changed !